articles/migrations/guides/liveid-api-deprecation.md

```diff
@@ -0,0 +1,135 @@
+ 
+
+***OIDC Profile***
+
+| Field  |  Live SDK |  Microsoft Graph  |
+|--------|---------------|------------------|
+|  picture | Returned a URL for the user picture | Auth0 will build a URL that will return a default picture for the user based on their initials. To get the actual image you need to [download it using Microsoft Graph](https://docs.microsoft.com/en-us/graph/api/profilephoto-get?view=graph-rest-1.0)|
+| locale | Returned a string in the format en_US | Not available. |
+
+***Raw Profile***
+
+The raw user profile that can be obtained calling the [Get User By Id Endpoint](https://auth0.com/docs/api/management/v2#!/Users/get_users_by_id) has several differences when using Live SDK or Microsoft Graph.
+
+The JSON below shows the fields that are in the Live SDK profile but are not present, or are different, in the Microsoft Graph one:
+
+```js
+{
+    "locale": "en_ZA",
+    "work": [],
+    "emails": [
+        "john.doe@windowslive.com"
+    ],
+    "addresses": {
+        "personal": {
+            "street": null,
+            "street_2": null,
+            "city": null,
+            "state": null,
+            "postal_code": null,
+            "region": null
+        },
+        "business": {
+            "street": null,
+            "street_2": null,
+            "city": null,
+            "state": null,
+            "postal_code": null,
+            "region": null
+        }
+    },
+    "phones": {
+        "personal": null,
+        "business": null,
+        "mobile": null
+    },
+    "nickname": "john.doe@windowslive.com",
+   ]
+}
+```
+
+This other one shows the fields that are in the Microsoft Graph profile but are not present, or a different, in the Live SDK one one:
+
+```js
+{
+    "strategy_version": 2,
+    "displayName": "John Doe",
+    "userPrincipalName": "john.doe@windowslive.com",
+    "businessPhones": [],
+    "nickname": "john.doe",
+}
+```
+
+Key differences are:
+
+- `strategy_version` has the value `1` when the user last logged-in with Live Connect, or `2` when they last logged in with Azure AD.
+- `nickname` as a different format. 
+- `locale`, `work`, `emails` array, `addresses`, `phones` are present in Live SDK but not in Microsoft Graph's.
+- `displayName`, `userPrincipalName`, `businessPhones` array are present in Microsoft Graph but not in Live SDK.
+
+If a user that previously logged in with Live Connect and then logged in with Azure AD, the profiles will be merged, so it will have the content of the Live ID one plus the new fields in Microsoft Graph.
+
+The `strategy_version` will be set to '2' for users that last logged in with Azure AD, to '1' for users that logged in with Live SDK recently, and will not be present for users that did not login recently. You can use this field to better interpret the profile content.
```

Comment:
The strategy_version field will be...with AzureAD, and to '1' for users that have logged in with Live Connect recently. This field will not be present for users that have not logged in recently.